concepts/05 Widgets/Floating Action Button/00 Getting Started with Floating Action Button/10 Single Action.md

```diff
@@ -6,342 +6,4 @@ To create a single-action FAB, add one [SpeedDialAction](/api-reference/10%20UI%
 
```

Comment:
Line 5: `Setting other options is not required, but we also specify a [hint](/api-reference/10%20UI%20Widgets/Widget/1%20Configuration/hint.md '/Documentation/ApiReference/UI_Widgets/dxSpeedDialAction/Configuration/#hint').`

I'd like to avoid the -ing form ("Setting") here. Can we say: "You do not need to set other options, but you should specify a..."